sdk/search/Azure.Search.Documents/samples/Sample02_Service.md

```diff
@@ -118,8 +118,18 @@ when indexing to better fit your index. You might do this, for example, if you h
 indexing data from different data sources with different schemas and data.
 
 ```C# Snippet:Azure_Search_Tests_Samples_CreateIndexerAsync_CreateDataSourceConnection
+// Set SearchIndexerClient options
```

Comment:
Maybe say "Create SearchIndexerClient options", or you can even leave the comment out.